inbm/cloudadapter-agent/cloudadapter/cloud/cloud_builders.py

```diff
@@ -143,6 +143,7 @@ def build_messenger_with_config(config: Dict[str, Any]):
     telemetry = config.get("telemetry")
     attribute = config.get("attribute")
     event = config.get("event")
+    update = config.get("update")
```

Comment:
I think this should be `node_update = config.get("[maybe nodeUpdate or node_update?]")`